arrayMethod.js

```diff
@@ -0,0 +1,72 @@
+const arr = [33,56,78,66,22,88,66]
+let languages = ['C++', 'Java', 'Html', 'Python', 'C'];
+let webDvlop = ["HTML", "CSS", "JS", "Bootstrap"];
+
+//////////// entries = return key value pair of every index
+let checkEntries = languages.entries();
+for(value of checkEntries ){
+    console.log("entries",value);
+}
+
+///////////// find = returns first element which passes the condition 
+let found = arr.find(function (element) {
+    return element > 10;
+});
+let indexOf = arr.indexOf(found);  /// returns the indexof element
+console.log("index of",indexOf)
+
+/// Filter = it return the array of all element that passes the condition
+
+let filtered = arr.filter(isEven);
+console.log("filter",filtered)
+
+
+
+///////////////////////////some method :- pass a test and return a boolean
+function isEven(value) {
```

Comment:
No need to add multiple forward slash "/" only two is sufficient to comment any line.